src/pages/index.tsx

```diff
@@ -93,48 +92,132 @@ const Index: VFC<Props> = ({ data, error }) => {
   );
 };
 
-function HeroSection() {
+const EVENT_DATE = new Date("2021-12-25T15:00:00Z");
+function calculateDistance() {
+  return EVENT_DATE.getTime() - new Date().getTime();
+}
+const button = tw` text-black background[#FFFFFF] text-xs transition-all hover:(filter saturate-150) py-sm px-md mt-lg  rounded-lg  md:(py-sm px-lg mt-2xl text-sm) xl:( py-sm px-xl rounded-xl)`;
+
+function Christmas2021LandingPage() {
   const { t } = useTranslation();
+  const [distance, setDistance] = useState<number>(calculateDistance);
+
+  const { days, hours, minutes, seconds } = getCountdown(distance);
+
+  useEffect(() => {
+    const timerId = setInterval(() => setDistance(calculateDistance()), 1000);
+
+    return () => clearInterval(timerId);
+  }, []);
 
   return (
-    <section
-      tw="col-span-full grid grid-cols-12 content-start xl:-mt-24"
-      aria-labelledby="hero-title"
-    >
-      <header tw="text-center grid-p-sm  xl:(col-start-2 col-end-6 self-center z-10 width[150%] text-left)">
-        <h1
-          id="hero-title"
-          tw="text-2xl text-primary font-black md:text-4xl lg:text-5xl"
+    <section css={"width: 100vw"}>
```

Comment:
This causes an overflow on desktop.
```suggestion
    <section tw="col-span-full">
```